### PR TITLE
23 generating track

### LINKED ISSUE #23: Generating more track
The track has to be infinitely long, after starting, there has to be generated more track as the player moves along it.

### PR DESCRIPTION
**Pull request by MathiasMeuleman**
Will close #23 

Please review

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/gui/SplashScreen.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
  *
  * A splashcreen that show a "Press any key to continue", also starts the entire application.
  */
-public class SplashScreen extends Application {
+public class  SplashScreen extends Application {
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/SplashScreen.java]: Extra space? o.o why XD

**Comment by mpsijm** [comment]
Okay, apart from a bunch of checkstyle stuff (which you should run for yourself, it has no use listing them here) this branch is ready to be merged :) Nice work! :D

**Comment by MathiasMeuleman** [comment]
Any suggestions on how to fix the mgicnumbers? Kind of the point of this randomness in the track generation.

**Comment by mpsijm** [comment]
Things that have to do with the track width could already use that constant, for example:
`nextInt(WIDTH + 1)`
`lane = (lane + 1) % WIDTH`
And for the magic numbers in the tests: just some local final ints and you'll be fine :3

**Comment by mpsijm** [comment]
And about `private int minLaneLength` and its brother: maybe they should be final constants? Then you can also make the `nextInt(10)` a constant :) 
(It's hard to review code on your phone btw, so sorry if I didn't get the field names xD)

**Comment by MathiasMeuleman** [comment]
I fixed most of them, there are a few I cant figure out:
 - Line is longer than 80 (because of massive indent)
 - if statements are not styled correctly, brackets should be on the same line, but when I put the brackets on the same line, checkstyle thinks the brackets need a new line??
 - '*' is not followed/preceeded by whitespace?

**Comment by mdingjan** [comment]
@MathiasMeuleman, we already did a peer review. ;)
So if you're done with those strange checkstyle things ;P, then we can merge this branch :)

**Comment by MathiasMeuleman** [comment]
Thats the point.. Don't know what to do with them. Also, GitHub keeps complaining about a conflict in my branch, but I can't spot it

**Comment by mpsijm** [comment]
I see the checkstyle warnings have been fixed, will merge :)

**Pull request merged**